dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -137,6 +137,13 @@ function create_dasd_partitions {
     local ignore_cmd=0
     local ignore_cmd_once=0
     local cmd
+
+    # We resize first partition to enfore rewrite of the current
+    # partition table updated to the actual disk geometry. This is
+    # to circumvent the fdasd limitation of not being capable to
+    # expand the partition table up to the disk size bsc#1209247
+    parted --script --machine "${disk_device}" resizepart 1
```

Comment:
shouldn't this code part also be s390 specific ?

Reply:
this is only used for dasd partitions (is dasd existing beyond s390?). I agree this is not explicitly called only for s390, but this method is only for dasd partitions which are only detected by `parted` in `get_partition_table_type`.

We can add an arch specific path here, but feels redundant to me. Also the fdasd tool called down the line is also only added for s390 arch (see the `module-setup.sh`) so this method was already not functional for any arch other than s390. Here there is (and there was already) the assumption that partition type dasd matches s390 arch, I think this is a reasonable assumption. I am not familiar with s390 specifics though.

Reply:
> this is only used for dasd partitions (is dasd existing beyond s390?).

dasd is only exists as a disk type for s390x.

Reply:
Sorry I overlooked from reading the patch that this code is in the ```create_dasd_partitions``` which is already exclusive to s390 and the DASD handling. I agree all good, sorry for the noise